### ISSUE TITLE
2 hop link

### ISSUE THREAD

**Issue by tadashi-aikawa** [tadashi-aikawa/obsidian-another-quick-switcher#155]
If there are relations as follows,

```mermaid
flowchart LR
    D[Dog] --> A[Animal]
    C[Cat] --> A
    O[Owl] --> A
```

`2 hop link` from the "Dog" is as follows.

```mermaid
flowchart LR
    D[Dog]:::focus --> A[Animal]
    D[Dog]:::focus --> C[Cat]
    D[Dog]:::focus --> O[Owl]
    
    classDef focus fill:#f96
```

More complicated example.

```mermaid
flowchart LR
    Dog:::focus --> Animal
    Dog --> Masaru
    Zagitova --> Masaru
    Masaru --> Akita-inu
    Cat --> Animal
    Owl --> Animal
    Tama --> Cat
    Mike --> Cat
    Animal --> Zoo
    Zoo --> Animal
    Mahjong --> Zoo
    Animal --> Coffee
    classDef focus fill:#f96
```

`2 hop link` from the "Dog" is as follows.

```mermaid
flowchart LR
    Dog:::focus --> Animal:::focus
    Dog --> Masaru:::focus
    Zagitova:::focus --> Masaru
    Masaru --> Akita-inu
    Cat:::focus --> Animal
    Owl:::focus --> Animal
    Tama --> Cat
    Mike --> Cat
    Animal --> Zoo
    Zoo:::focus --> Animal
    Mahjong --> Zoo
    Animal --> Coffee
    classDef focus fill:#f96
```

**Comment by tadashi-aikawa** [comment]
![image](https://user-images.githubusercontent.com/9500018/224550705-9b001ceb-5f0e-4ad5-8209-19951f0b1545.png)

**Comment by tadashi-aikawa** [comment]
Released in [v8.7.0-beta1](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/releases/tag/8.7.0-beta1) 🚀

**Comment by tadashi-aikawa** [comment]
Released in [v8.7.0](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/releases/tag/8.7.0) 🚀
